tests/unit/api/test_bootstrap.py

```diff
@@ -134,6 +135,14 @@ def test_post_bootstrap(self, test_client, monkeypatch):
             mocked__check_bootstrap_status,
         )
 
+        fake_time = datetime.datetime(2019, 6, 16, 9, 5, 1)
+        fake_datetime = pretend.stub(
+            now=pretend.call_recorder(lambda: fake_time)
+        )
```

Comment:
At the end of the test, you want to add a line testing the `fake_datetime.now` calls.
Check how we assert calls for other call recorders in this function.